helper-cli/src/main/kotlin/commands/GetPackageLicensesCommand.kt

```diff
@@ -33,11 +33,12 @@ import org.ossreviewtoolkit.helper.utils.PackageConfigurationOption
 import org.ossreviewtoolkit.helper.utils.createProvider
 import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.LicenseFinding
+import org.ossreviewtoolkit.model.ScanResult
```

Comment:
Maybe squash to the previous commit and do this for all `helper-cli` commands at once?

Reply:
I'd rather keep them separate.